sample-lnd.conf

```diff
@@ -975,10 +975,13 @@ litecoin.node=ltcd
 
 [wtclient]
 
-; Activate Watchtower Client. To get more information or configure watchtowers
+; (Deprecated) Activate Watchtower Client. To get more information or configure watchtowers
```

Comment:
This text should be synced with the flag description, same with the `Deactivate` flag below.